### PR TITLE
Apply min-preludes to more tests (part 7)

### PR DESCRIPTION
**Pull request by danakj**
This drops file_test runtime from about 3s to 2.5s on my machine, which is now ~30% faster than before #5653 slowed things down by adding a lot of stuff to the production prelude.

### CODE REVIEW

**Comment by danakj** [comment]
based on https://github.com/carbon-language/carbon-lang/pull/5691, first commit is `min-preludes7`

File: toolchain/check/testdata/function/call/empty_struct.carbon [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/none.carbon
 // TODO: Add ranges and switch to "--dump-sem-ir-ranges=only".
 // EXTRA-ARGS: --no-prelude-import --dump-sem-ir-ranges=if-present
```
Comment (jonmeow) [inline toolchain/check/testdata/function/call/empty_struct.carbon]:
```suggestion
// EXTRA-ARGS: --dump-sem-ir-ranges=if-present
```

File: toolchain/check/testdata/function/call/empty_tuple.carbon [hunk 2]
```diff
@@ -2,6 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/none.carbon
 // TODO: Add ranges and switch to "--dump-sem-ir-ranges=only".
 // EXTRA-ARGS: --no-prelude-import --dump-sem-ir-ranges=if-present
```
Comment (jonmeow) [inline toolchain/check/testdata/function/call/empty_tuple.carbon]:
```suggestion
// EXTRA-ARGS: --dump-sem-ir-ranges=if-present
```

File: toolchain/check/testdata/function/call/fail_explicit_self_param.carbon [hunk 3]
```diff
@@ -2,6 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/none.carbon
 // TODO: Add ranges and switch to "--dump-sem-ir-ranges=only".
 // EXTRA-ARGS: --no-prelude-import --dump-sem-ir-ranges=if-present
```
Comment (jonmeow) [inline toolchain/check/testdata/function/call/fail_explicit_self_param.carbon]:
```suggestion
// EXTRA-ARGS: --dump-sem-ir-ranges=if-present
```

File: toolchain/check/testdata/function/call/fail_runtime_implicit_param.carbon [hunk 4]
```diff
@@ -2,6 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/none.carbon
 // TODO: Add ranges and switch to "--dump-sem-ir-ranges=only".
 // EXTRA-ARGS: --no-prelude-import --dump-sem-ir-ranges=if-present
```
Comment (jonmeow) [inline toolchain/check/testdata/function/call/fail_runtime_implicit_param.carbon]:
```suggestion
// EXTRA-ARGS: --dump-sem-ir-ranges=if-present
```

File: toolchain/check/testdata/function/call/params_zero.carbon [hunk 5]
```diff
@@ -2,6 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/none.carbon
 // TODO: Add ranges and switch to "--dump-sem-ir-ranges=only".
 // EXTRA-ARGS: --no-prelude-import --dump-sem-ir-ranges=if-present
```
Comment (jonmeow) [inline toolchain/check/testdata/function/call/params_zero.carbon]:
```suggestion
// EXTRA-ARGS: --dump-sem-ir-ranges=if-present
```

File: toolchain/check/testdata/function/generic/forward_decl.carbon [hunk 6]
```diff
@@ -2,7 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/convert.carbon
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/none.carbon
 // TODO: Add ranges and switch to "--dump-sem-ir-ranges=only".
 // EXTRA-ARGS: --no-prelude-import --dump-sem-ir-ranges=if-present
```
Comment (jonmeow) [inline toolchain/check/testdata/function/generic/forward_decl.carbon]:
```suggestion
// EXTRA-ARGS: --dump-sem-ir-ranges=if-present
```

File: toolchain/check/testdata/impl/lookup/specific_args.carbon [hunk 7]
```diff
@@ -2,6 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/none.carbon
 // TODO: Add ranges and switch to "--dump-sem-ir-ranges=only".
 // EXTRA-ARGS: --no-prelude-import --dump-sem-ir-ranges=if-present
```
Comment (jonmeow) [inline toolchain/check/testdata/impl/lookup/specific_args.carbon]:
```suggestion
// EXTRA-ARGS: --dump-sem-ir-ranges=if-present
```

**Review by jonmeow — APPROVED** [review]
Basically LG, but don't forget to remove `--no-prelude-import` when you're switching off of it.

**Comment by danakj** [comment]
> Basically LG, but don't forget to remove `--no-prelude-import` when you're switching off of it.

Woops, I missed quite a few, thanks
